dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/api/CommonSDKServlet.java

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat.apps.api;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.common.Clock;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.common.InstrumentationScopeInfo;
+import io.opentelemetry.sdk.common.InstrumentationScopeInfoBuilder;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.resources.ResourceBuilder;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.servlet.annotation.WebServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/common")
+@ApplicationScoped // Make this a bean so that there's one bean in the archive, otherwise CDI gets disabled and @Inject doesn't work
+public class CommonSDKServlet extends FATServlet {
+
+    /**
+     * Very simple test that we can use Clock.nanoTime()
+     * {@link Clock}
+     *
+     * @throws InterruptedException
+     */
+    @Test
+    public void testClockNanoTime() throws InterruptedException {
+        Clock clock = Clock.getDefault();
+        long startNano = clock.nanoTime();
+        int sleepNano = 100000;
+        Thread.sleep(0, sleepNano);
+        long endNano = clock.nanoTime();
+        long durationNano = endNano - startNano;
+        assertTrue(durationNano >= sleepNano);
+        assertTrue(durationNano < (10 * sleepNano));
```

Comment:
I think I'd prefer to remove this assert, we've definitely seen build systems pause for more than a second.

Reply:
I think I misread. 100000ns is only 0.1ms, so I think we're actually checking that elapsed duration is <1ms, which is way too short.

I would be tempted to replace any nanosecond time constants and conversions with `Duration`. (i.e. `sleepNano = Duration.of(100, MILLIS).toNanos();`